### ISSUE TITLE
Error when executing `npm run examples`

### ISSUE THREAD

**Issue by PeterNgTr** [codecept-js/ui#35]
I got this error when trying to perform `npm run examples`

```sh
Thanhs-MacBook-Pro:ui thanhnguyen$ npm run examples

> @codeceptjs/ui@0.2.1 examples /Users/thanhnguyen/Desktop/ui
> cd node_modules/@codeceptjs/examples && ../../../bin/codecept-ui.js

/Users/thanhnguyen/Desktop/ui/lib/utils/mkdir.js:8
      throw e;
      ^

Error: ENOENT: no such file or directory, mkdir '/Users/thanhnguyen/Desktop/ui/node_modules/@codeceptjs/examples/output/.ui'
    at Object.mkdirSync (fs.js:764:3)
    at mkdir (/Users/thanhnguyen/Desktop/ui/lib/utils/mkdir.js:5:8)
    at Object.<anonymous> (/Users/thanhnguyen/Desktop/ui/lib/model/scenario-status-repository.js:7:1)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Users/thanhnguyen/Desktop/ui/lib/codeceptjs/realtime-reporter.helper.js:3:34) {
  errno: -2,
  syscall: 'mkdir',
  code: 'ENOENT',
  path: '/Users/thanhnguyen/Desktop/ui/node_modules/@codeceptjs/examples/output/.ui'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @codeceptjs/ui@0.2.1 examples: `cd node_modules/@codeceptjs/examples && ../../../bin/codecept-ui.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @codeceptjs/ui@0.2.1 examples script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/thanhnguyen/.npm/_logs/2019-12-03T16_39_52_154Z-debug.log
```

**Comment by DavertMik** [comment]
Did you update @examples to 1.1.0 ?

**Comment by PeterNgTr** [comment]
@DavertMik I tried yesterday but no luck.

**Comment by kaflan** [comment]
https://stackoverflow.com/questions/28498296/enoent-no-such-file-or-directory-on-fs-mkdirsync

**Comment by kaflan** [comment]
<img width="1189" alt="Screenshot 2019-12-04 at 16 13 39" src="https://user-images.githubusercontent.com/3959504/70149559-09f07f00-16b1-11ea-9e45-fb3076247952.png">

**Comment by PeterNgTr** [comment]
@kaflan I saw the `output` folder is [there](https://github.com/codecept-js/examples/tree/master/output) but not sure why we installed via npm, that folder is not showing.

**Comment by DavertMik** [comment]
Should be fixed in examples in 1.1.1. :)
